CMakeLists.txt

```diff
@@ -339,8 +339,8 @@ endif()
 #
 ExternalProject_Add(prometheus-cpp
   PREFIX prometheus-cpp
-  URL "https://github.com/jupp0r/prometheus-cpp/archive/v1.0.1.tar.gz"
-  URL_HASH SHA256=593e028d401d3298eada804d252bc38d8cab3ea1c9e88bcd72095281f85e6d16
+  GIT_REPOSITORY "https://github.com/jupp0r/prometheus-cpp.git"
+  GIT_TAG "76470b3ec024c8214e1f4253fb1f4c0b28d3df94"
```

Comment:
Why not provide a version tag instead of commit hash?

Reply:
I think commit hash could be more reliable, because the commit under a tag could be modified while it is (almost) impossible to have the same hash on two different commits.
https://stackoverflow.com/questions/21459540/add-new-commit-to-the-existing-git-tag/50454106#50454106

Reply:
A second through, since we are already using version tags for other third-party projects, if we do not need the extra safety against inadvertent code changes, I think version tags will be clearer.

Reply:
I would prefer version tags. The commit hash obscures which version of the library is being used. 
It will be better to build against version tag. I am assuming that they might want to move version tag for a strong case.